src/dmd/mtype.d

```diff
@@ -5966,7 +5970,7 @@ extern (C++) final class TypeEnum : Type
         return MATCH.nomatch;
     }
 
-    override Type toBasetype()
+    final Type toBasetype2()
```

Comment:
Maybe toMemtype?  I'm not bothered either way.  Also `extern(D)` so that it is not exported to C++ please.

Reply:
As this is now a unique function, it would also be good to add some Ddoc about what it does.

Reply:
I did the latter.